shared/people/todo/container.tsx

```diff
@@ -281,15 +282,15 @@ const VerifyAllEmailConnector = connect(
         ...(meta
           ? [
               {
-                label: hasRecentVerifyEmail ? `Resend the verification email` : 'Verify',
+                label: hasRecentVerifyEmail ? `Send the verification email again` : 'Verify',
```

Comment:
This is very verbose. Can we say `Resend verification` or `Resend`? Spills out of the button on iPhone 5S, it likely gets cut off mid-word on other screen sizes.
<img width="315" alt="image" src="https://user-images.githubusercontent.com/11968340/63178901-f2fe2200-c018-11e9-876c-a7b32849e630.png">

Reply:
I thought the non-resend case used to just be "verify" - that seems simplest to me and also doesn't flow off the screen.

Reply:
"again" is missing

Reply:
switching this to "Verify again"

Reply:
oh I missed the again. Maybe "Retry verification"? "verify again" sounds to me like you've already verified it successfully once.